### PR TITLE
Rename Email Command Params for better compatiblity

### PR DESCRIPTION
**Pull request by riensen**
Users are complaining that Auto-GPT is using the commands with different parameter names.
This is due to how LLMs work, they choose the most likely word.
I am changing here the params such that Auto-GPT is more likely to use the same param name.
I have also included some other smaller doc and error handling improvements.

### CODE REVIEW

File: src/autogpt_plugins/email/__init__.py [hunk 1]
```diff
@@ -57,6 +58,10 @@ def post_prompt(self, prompt: PromptGenerator) -> PromptGenerator:
                 },
                 send_email_with_attachment,
             )
+        else:
+            print(
+                Fore.RED + f"{self._name} - {self._version} - Email plugin not loaded, because EMAIL_PASSWORD or EMAIL_ADDRESS were not set in env."
```
Comment (ntindle) [inline src/autogpt_plugins/email/__init__.py:63]: As part of this, would you mind adding a utils folder with a logging file inside that exports a logger? Long term we will inject it into the plugins env but for now getting everyone off print seems like a good start

File: src/autogpt_plugins/email/email_plugin/email_plugin.py [hunk 2]
```diff
@@ -23,29 +25,29 @@ def getPwd():
...
 def send_email_with_attachment_internal(
-    recipient: str, subject: str, message: str, attachment_path: str, attachment: str
+    to: str, title: str, message: str, attachment_path: str, attachment: str
 ) -> str:
     """Send an email
 
     Args:
-        recipient (str): The email of the recipients
-        subject (str): The subject of the email
+        to (str): The email of the tos
```
Comment (ntindle) [inline src/autogpt_plugins/email/email_plugin/email_plugin.py:49]: The tos?

**Review by ntindle — APPROVED** [review]

File: src/autogpt_plugins/email/__init__.py (hunk 1 above)
Comment (riensen) [reply src/autogpt_plugins/email/__init__.py:63]: The logger is acutally in auto-gpt. So we should use that one. But that is a discussion we still have to have - how we import autogpt here. autogpt is not a lib on pip or anywhere, so we can not easily import it in a supported way.

File: src/autogpt_plugins/email/email_plugin/email_plugin.py (hunk 2 above)
Comment (riensen) [reply src/autogpt_plugins/email/email_plugin/email_plugin.py]: done

**Review by ntindle — APPROVED** [review]

**Pull request merged**